<launch>
    <node pkg="robot_localization" type="navsat_transform_node" name="navsat_transform">
        <param name="zero_altitude" value="true"/>
	<param name="broadcast_utm_transform" value="true"/>
	<param name="published_frame" value="map"/>
	<param name="yaw_offset" value="1.5708"/>
        <remap from="imu/data" to="/zed2/zed_node/imu/data"/>
        <remap from="gps/fix" to="/gps/fix"/>
        <remap from="odom" to="/odometry/filtered"/>
	<param name="magnetic_declination_radians" value="0.15708"/>
    </node>
</launch>

